src/topology/channel.c

```diff
@@ -60,7 +60,7 @@ static const struct map_elem channel_map[] = {
 };
 
 
-static int lookup_channel(const char *c)
+int lookup_channel(const char *c)
```

Comment:
The naming of the non-static function should follow 'tplg_' prefix (see tplg_local.h).